src/TrackPanelAx.cpp

```diff
@@ -466,7 +466,8 @@ wxAccStatus TrackPanelAx::GetState( int childId, long* state )
    }
    else     // childId == wxACC_SELF
    {
-      *state = wxACC_STATE_SYSTEM_FOCUSABLE + wxACC_STATE_SYSTEM_FOCUSED;
+      // let wxWidgets use a standard accessible object for the state
+      return wxACC_NOT_IMPLEMENTED;
```

Comment:
I'm not familiar with that code, and also I cannot hit a breakpoint here that'd give me more context. Would I need a narrator for that, I suppose? @DavidBailes if you're okay with it let's just proceed with this one to QA, provided you've added a QA checklist in the description.

Reply:
Yes, TrackPanelAx::GetState() only gets called if a screen reader or one or the accessibility testing tools are running. The relevant code for how wxACC_NOT_IMPLEMENTED is handled is pasted below.
Note sure a QA checklist is needed - it's pretty obvious whether on opening, the dialog is automatically read by Narrator and NVDA.
I'm happy for it to go straight to QA.

```
STDMETHODIMP wxIAccessible::get_accState ( VARIANT varID, VARIANT* pVarState)
{
    wxLogTrace(wxT("access"), wxT("get_accState"));
    wxASSERT( ( m_pAccessible != NULL ) || ( m_bQuiescing == true ) );
    if (!m_pAccessible)
        return E_FAIL;

    if (varID.vt != VT_I4 || varID.lVal < 0)
    {
        wxLogTrace(wxT("access"), wxT("Invalid arg for get_accState"));
        return E_INVALIDARG;
    }

    long wxstate = 0;

    wxAccStatus status = m_pAccessible->GetState(varID.lVal, & wxstate);
    if (status == wxACC_FAIL)
        return E_FAIL;
    if (status == wxACC_INVALID_ARG)
        return E_INVALIDARG;

    if (status == wxACC_NOT_IMPLEMENTED)
    {
        // Try to use child object directly.
        if (varID.lVal > 0)
        {
            IAccessible* childAccessible = GetChildAccessible(varID.lVal);
            if (childAccessible)
            {
                varID.lVal = 0;
                HRESULT hResult = childAccessible->get_accState(varID, pVarState);
                childAccessible->Release();
                return hResult;
            }
            else if (m_pAccessible->GetIAccessibleStd())
                return m_pAccessible->GetIAccessibleStd()->get_accState(varID, pVarState);
        }
        else if (m_pAccessible->GetIAccessibleStd())
            return m_pAccessible->GetIAccessibleStd()->get_accState (varID, pVarState);
    }
    else
    {
        LONG state = wxConvertToWindowsState(wxstate);
        pVarState->lVal = state;
        pVarState->vt = VT_I4;
        return S_OK;
    }
    return E_NOTIMPL;
}
```

Reply:
Ok.

Reply:
> provided you've added a QA checklist in the description

I've added a note for QA to test with both Narrator and NVDA.